website/notifications/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'Saman'
```

Comment:
You'll still need this file in order to make `website.notifications` a proper module.